fv/latency_test.go

```diff
@@ -160,6 +160,12 @@ var _ = Context("Latency tests with initialized Felix and etcd datastore", func(
 		It("with allow-all should have good latency", func() {
 			meanRtt, out := w[0].LatencyTo(w[1].IP, w[1].DefaultPort)
 			_, err := fmt.Fprintf(resultsFile, "allow-all: %v\n", meanRtt)
+
+			if meanRtt > 10 {
+				log.Info("Sleeping.....")
+				time.Sleep(1000000 * time.Second)
+			}
+
```

Comment:
Presumably this was debugging and should be removed now.

Reply:
Yes, d'oh.